docs/sphinx/api.rst

```diff
@@ -39,6 +39,9 @@ ServiceState
 ------------
 
 .. autoclass:: yapapi.services.ServiceState
+    :members: pending, starting, running, stopping, terminated,
+        start, ready, stop, terminate, restart,
+        lifecycle, error_or_stop, AVAILABLE, instance
```

Comment:
I think it would be much better to have **only** 
* the five states here (i.e. the first line).
* `AVAILABLE`

in the API Reference.

Mixing in some methods that are only used internally will confuse readers.
Also we don't really want anyone to call them directly, I guess?